client/store/modules/editor/getters.js

```diff
@@ -20,9 +21,23 @@ export const contentContainers = (_state, getters, { repository }) => {
   }, {});
 };
 
+export const contentContainers = (_state, getters, { repository }) => {
+  const { rootContainerGroups } = getters;
+  const { items: activities } = repository.activities;
+  const parents = flatMap(rootContainerGroups);
+  const children = findChildren(activities, parents);
```

Comment:
Consider putting parents as first argument (You are trying to find children of those parents so it feels like that's the primary param)